pkg/kubectl/cmd/drain.go

```diff
@@ -282,16 +283,28 @@ func (o *DrainOptions) getController(sr *api.SerializedReference) (interface{},
 	return nil, fmt.Errorf("Unknown controller kind %q", sr.Reference.Kind)
 }
 
+func newSerializedControllerRef(pod api.Pod) *api.SerializedReference {
+	return &api.SerializedReference{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "v1",
+			Kind:       "SerializedReference",
+		},
+		Reference: api.ObjectReference{ //currently only set based on first controller
+			APIVersion: pod.GetOwnerReferences()[0].APIVersion,
```

Comment:
noted! i thought a pod has only one owner like cronjob and job, and decided to have a try with `pod.GetOwnerReferences()[0]`. i will use another approach.